TruckMuncher/VendorMapViewController.swift

```diff
@@ -196,16 +197,45 @@ class VendorMapViewController: UIViewController, MKMapViewDelegate, CLLocationMa
     }
     
     func requestServingMode (isServing: Bool) {
+        var itemsOutOfStock = getOutOfStockItems()
+        if itemsOutOfStock.count > 0 && isServing == true {
+            var message = String()
+            for item in itemsOutOfStock {
+                message += "- " + (item as RMenuItem).name + "\n"
```

Comment:
That is true. However, if a truck has that many things out of stock, they probably won't be in business very long.